classes/Tools.php

```diff
@@ -2447,13 +2459,12 @@ public static function generateHtaccess($path = null, $rewrite_settings = null,
         }
 
         foreach ($domains as $domain => $list_uri) {
-            $physicals = array();
             foreach ($list_uri as $uri) {
                 fwrite($write_fd, PHP_EOL . PHP_EOL . '#Domain: ' . $domain . PHP_EOL);
                 if (Shop::isFeatureActive()) {
-                    fwrite($write_fd, 'RewriteCond %{HTTP_HOST} ^' . $domain . '$' . "\n");
+                    fwrite($write_fd, 'RewriteCond %{HTTP_HOST} ^' . $domain . '$' . PHP_EOL);
                 }
-                fwrite($write_fd, 'RewriteRule . - [E=REWRITEBASE:' . $uri['physical'] . ']' . "\n");
+                fwrite($write_fd, 'RewriteRule . - [E=REWRITEBASE:' . $uri['physical'] . ']' . PHP_EOL);
 
                 // Webservice
                 fwrite($write_fd, 'RewriteRule ^api$ api/ [L]' . "\n\n");
```

Comment:
PHP_EOL

Reply:
I will not change every \n in this class :P 
It was just a fast replace for my needs =)

Reply:
Okay :)